Makefile

```diff
@@ -88,13 +88,13 @@ test:	rclone test_all
 
 # Quick test
 quicktest:
-	RCLONE_CONFIG="/notfound" go test $(LDFLAGS) $(BUILDTAGS) ./...
+	RCLONE_CONFIG="/notfound" go test $(BUILDTAGS) ./...
```

Comment:
This is necessary because Golang reuses the tests cache only if the flags remain the same and having the dynamic version as part of the flags was causing the cache to never be re-used.

Reply:
Nice. Can you do the same thing to the other tests in the makefile which are next to this one.

Reply:
I did apply the same changes to the `racequicktest` & `compiletest` too below. I left `test_all` as that's not run on the Github CI anyway. I don't think I missed anything?